openvox.gemspec

```diff
@@ -1,6 +1,15 @@
+# When installed, this gemspec ends up the specifications dir and we should look for
+# the library at the installed path because Puppet ends up loading it when invoked.
+# When we are building the gem from source, we instead look relative to where this gemspec
+# file lives. This is a little jank, but prevents us from having to update the version in
+# two different places.
+installed_path = '/opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/version'
+version_path = File.exist?("#{installed_path}.rb") ? installed_path : 'lib/puppet/version'
+require_relative "#{version_path}"
+
```

Comment:
I do not recall the exact details, but IIRC, this gemspec is installed "as is" while in a regular installation (building a gem with gem build, and installing it), the installed gemspec has been generated from the metadata collected in the gemspec at build time.

I think I tried to do something similar (read the version from version file instead of hardcoding), but was probably missing these bits.  I wonder if/how we can avoid this workaround.